robot/EDA/tests/browser/settings/accounts_contacts.robot

```diff
@@ -59,4 +59,48 @@ Verify disable preferred phone enforcement shows and hides as expected
     Verify checkbox value
     ...                       Enable Enhanced Preferred Phone Functionality=false
     Verify disable preferred phone enforcement displayed
-    ...                       Disable Preferred Phone Enforcement=false
\ No newline at end of file
+    ...                       Disable Preferred Phone Enforcement=false
+
+Validate all checkboxes can retain value on save in accounts and contacts settings
+    [Documentation]         Validates that all checkbox fields in accounts and contacts settings
+    ...                     page retains value after clicking on save button. The status of checkbox
+    ...                     is true if checked and false if unchecked
+    [tags]                  unstable        W-8620850       rbt:high
+    Click action button on EDA settings page    Edit
+    Set checkbox value
+    ...                 Disable Preferred Email Enforcement=true
+    ...                 Enable Enhanced Preferred Phone Functionality=true
+    ...                 Disable Preferred Phone Enforcement=true
```

Comment:
If this test case fails in between the execution, would the point of failure affect any other test cases in the future ? Just checking if adding any teardown steps (either now or at a later point) would help with inconsistencies.